dafni_cli/filtering.py

```diff
@@ -0,0 +1,105 @@
+from datetime import datetime
+from typing import Any, Callable, List, Union
+
+from dafni_cli.models.model import Model
+from dafni_cli.workflows.workflow import Workflow
+
+
+def filter_multiple(
+    filters: List[Callable[[Any], bool]], instances: List[Any], dictionaries: List[dict]
+) -> List[Any]:
+    """Filters a list of objects given a list of functions that must all return
+    True
+
+    Args:
+        filters (List[Callable[[Any], bool]]): List of filters that each
+                 receive a value from the list and should return true
+                 if that value should be in the filtered list or false
+                 otherwise.
+        values (List[Any]): List of values to filter
```

Comment:
What are the instances and dictionaries in this function and how do they relate to each other? I see they're zipped and looped through below

Reply:
I should have added them to the docstring, hopefully it is clearer now.